docker/step/Dockerfile

```diff
@@ -4,34 +4,24 @@ FROM ubuntu:18.04
 # The default source archive.ubuntu.com is busy and slow. We use the following source makes docker build running faster.
 COPY docker/dev/find_fastest_resources.sh /usr/local/bin/find_fastest_resources.sh
 
+ENV DEBIAN_FRONTEND=noninteractive
 RUN /bin/bash -c 'source find_fastest_resources.sh \
   && echo "Choose the fastest APT source ..." \
   && choose_fastest_apt_source \
   && echo "Choose the fastest PIP source ..." \
-  && choose_fastest_pip_source'
-
-RUN apt-get install -y openjdk-8-jre-headless wget unzip > /dev/null
-
-RUN wget -q http://docs-aliyun.cn-hangzhou.oss.aliyun-inc.com/assets/attach/119096/cn_zh/1557995455961/odpscmd_public.zip && \
-unzip -qq odpscmd_public.zip -d /usr/local/odpscmd && \
-ln -s /usr/local/odpscmd/bin/odpscmd /usr/local/bin/odpscmd && \
-rm -rf odpscmd_public.zip
-
-ADD build/step /usr/bin/
-ADD build/*.jar /opt/sqlflow/parser/
-
-ENV DEBIAN_FRONTEND=noninteractive
-RUN ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
-apt-get install -y tzdata > /dev/null && \
-dpkg-reconfigure --frontend noninteractive tzdata
-
-RUN apt-get install -y build-essential libmysqlclient-dev > /dev/null
-
-RUN apt-get install -y python3 python3-pip
-RUN ln -s /usr/bin/python3 /usr/bin/python
-RUN ln -s /usr/bin/pip3 /usr/bin/pip
-
-RUN bash -c "pip install numpy==1.16.1 \
+  && choose_fastest_pip_source' && \
```

Comment:
Can we move this long command into a bash file?

Reply:
I'd like to put scripts directly inside the Dockerfile, I like "flat is better than nested".